### DISCUSSION TITLE
Noob wanting to be spoon fed.

### DISCUSSION THREAD

**Discussion by lilteddyp** [chuot/rdio-scanner#106] [Q&A]
Looking to start up a personal stream for a local p25 system. MPSCS/Ingham County (https://www.radioreference.com/apps/db/?sid=100) to be exact. I've looked and found out I'll need 6 dongles. Would a Raspberry Pi 4 8gb handle a system like this? Or do I need to find an older desktop to run on? I don't know anything about coding, but love scanning, and Rdio Scanner is basically exactly what I was thinking I wanted to figure out. Stumbled across this and now am looking for any help available to get it started. I have a Uniden SDS100 that I have with me all the time, but the app idea is really cool, especially if I miss a call. I like the idea of the Pi, but just want it to run correctly and not have to constantly maintain it.

**Comment by chuot** [comment]
Rdio Scanner will run on a rpi 4b+ no problem.

As for Trunk Recorder, I haven't experienced it on a 4b+, but on a 3b+ I was able to run no more than 2 dongles directly attached to the rpi and no more than 10 recorders max. With 6 dongles, it would be advisable to run them on a powered usb hub. A usb hub is also better because RTL dongle gets very hot, which isn't great when connected directly to the rpi.

**Reply by photounit** [reply]
I use HP thin clients or small form factor PC's.  You can find them used on Ebay for a reasonable price and they are a nice self-contained solution. And depending on the model, significantly more powerful than a Rasberry pi which you will need for running 6 dongles in Trunk Recorder....

**Comment by rpdale** [comment]
You may want to go the AirSpy route... It has a much larger coverage so you wouldn't need nearly as many devices (I'm using one AirSpy and one RTL for the 857MHz freq to feed this system https://www.radioreference.com/apps/db/?sid=9099 )

https://airspy.com/airspy-r2/

Here's where you would start the process of getting the software to actually feed Rdio - https://github.com/robotastic/trunk-recorder

**Reply by lilteddyp** [reply]
I would need 2 airspys to cover my area. 6 SDRs is cheaper than 2 airspys lol. Thanks for the suggestion though. I wish it were that easy.

**Reply by rpdale** [reply]
Agreed - and that's the issue as I can't see any way that a RPI can deal with 6 SDRs. You'd probably need to go with the thin client route to run that many. That's why I gave it up when the county switched over instead of trying to make it work :)
